problems/removeVowels.js

```diff
@@ -6,8 +6,8 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+function removeVowels(str) {
+    return str.replace("A", "E" ,"I", "O", "U")
```

Comment:
For this problem you should be iterating through the chars of a str and building a new string if they're not one of the vowels.